Networking/Sources/Networking/Base/Endpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
+//  
+//
+//  Created by DariaMikots on 06.07.2022.
+//
+
+import Foundation
+
+public protocol Endpoint {
+    var baseURL: String { get }
+    var path: String { get }
+    var method: HTTPMethod { get }
+    var header: [String: String]? { get }
+    var body: [String: String]? { get }
```

Comment:
Why nillable dictionaries? Usually it's better to just make them empty if there are no values